src/test/kotlin/TestUtils.kt

```diff
@@ -1,18 +1,19 @@
-import io.hsar.wh40k.combatsimulator.model.AttackExecutor
+
+import com.nhaarman.mockito_kotlin.any
+import io.hsar.wh40k.combatsimulator.logic.actionoptions.*
 import io.hsar.wh40k.combatsimulator.model.MapPosition
 import io.hsar.wh40k.combatsimulator.model.UnitInstance
 import io.hsar.wh40k.combatsimulator.model.World
-import io.hsar.wh40k.combatsimulator.model.unit.Attribute
-import io.hsar.wh40k.combatsimulator.model.unit.BaseStat
-import io.hsar.wh40k.combatsimulator.model.unit.DerivedStats
-import io.hsar.wh40k.combatsimulator.model.unit.NumericValue
-import io.hsar.wh40k.combatsimulator.model.unit.Stats
+import io.hsar.wh40k.combatsimulator.model.unit.*
 import io.hsar.wh40k.combatsimulator.model.unit.Unit
+import org.mockito.Mockito
+import org.mockito.Mockito.*
```

Comment:
[Minor] Generally wanting to avoid star imports but this is a test so IDGAF really